features/is-analytics-solution-features/org.wso2.carbon.sp.solutions.is.analytics.feature/src/main/resources/siddhi-files/IS_ANALYTICS_SESSION_DATA.siddhi

```diff
@@ -1,8 +1,28 @@
-@App:name("IS_ANALYTICS_SESSION_DATA")
+@App:name("IS_ANALYTICS_SESSION_DATA")/*
+ * Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
 @App:description("Receiving Streams from IS and executing and analyzing statistics for session")
 
 -- Receving streams from IS
 --Defining Input Stream for receving streams from IS
+define trigger TriggerStream at '0 0 12 * * ?';
```

Comment:
Correct the comment, the comment is for the source not trigger stream. Add a comment for purging as well.